scripts/src/tools/git_repo.py

```diff
@@ -61,8 +62,8 @@ def update_and_clean(self):
 
     def checkout(self, branch_name):
         log_info("Checkout " + branch_name)
-        self.__repo.git.checkout(branch_name)
-        self.update_and_clean()
+        self.__repo.git.checkout(branch_name)        
+        #self.update_and_clean()
```

Comment:
I don't exactly remember if we commented the `update_and_clean()` for any special reason, but if not, can you check if the script stills works with this line uncommented?

I think it is a good practice to call this method each time there is a checkout.

Reply:
okay,I am checking with uncommented and let you know the result .

Reply:
After uncomment it is still working fine .